plugins/ui/src/deephaven/ui/components/button.py

```diff
@@ -111,7 +109,7 @@ def button(
         type: The behavior of the button when used in an HTML form.
         is_disabled: Whether the button is disabled.
         auto_focus: Whether the button should automatically receive focus when the page loads.
-        href: A URL to link to if elementType="a".
+        href: A URL to link to if element_type="a".
```

Comment:
Kind of confusing since we don't allow the user to actually specify `element_type="a"`. Maybe change up the wording?
```suggestion
        href: A URL to link to when the button is pressed.
```

Reply:
thanks, I missed that.